gigascience/gigadb-website
# Fix outstanding issues in home page

# Pull request for issue: #1535

This is a pull request for the following functionalities:

* Fix outstanding issues with home page http://gigadb.gigasciencejournal.com:9170/

## How to test?

* Navigate to http://gigadb.gigasciencejournal.com:9170/
* Run WAVE audit and confirm no "errors"

## How have functionalities been implemented?

* Added aria-label attribute to search input
* removed onclick handler and replaced with actual link
* Improved screen reader labels for metric headings

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
